mycore-mets/src/main/java/org/mycore/mets/iiif/MCRMetsIIIFPresentationImpl.java

```diff
@@ -50,6 +54,9 @@
 
     private static final Logger LOGGER = LogManager.getLogger();
 
+    public static final boolean STORE_METS_ON_GENERATE = MCRConfiguration2
+        .getOrThrow("MCR.Mets.storeMetsOnGenerate", Boolean::parseBoolean);
```

Comment:
Property naming: After "." start with capital letter - if possible. Also "Mets" is redundant: "MCR.METS.StoreXMLOnGenerate" would be better.

Reply:
Its not a new property. I just copied it from an other class.

Reply:
I created a new ticket for this:
https://mycore.atlassian.net/browse/MCR-2308